packages/model-viewer/src/features/scene-graph.ts

```diff
@@ -106,8 +106,10 @@ export const SceneGraphMixin = <T extends Constructor<ModelViewerElementBase>>(
     static Texture: Constructor<Texture>;
     static Image: Constructor<Image>;
 
-    private[$onUpdate]() {
-      this[$needsRender]();
+    private[$getOnUpdateMethod]() {
+      return () => {
+        this[$needsRender]();
```

Comment:
Discovered that change variant example was breaking because 'this' was undefined. I don't think it was ever being captured in a lambda, this fixes it.